app/models/approval.rb

```diff
@@ -16,6 +16,7 @@ class Approval < ActiveRecord::Base
 
   scope :individual, -> { where(type: 'Approvals::Individual') }
 
+  validates :proposal, presence: true, allow_blank: true
```

Comment:
Haha thanks. I was a little confused writing some of these, and figured allowing blank would be defeating the purpose. If I write a validation, and it causes the test to fail with the reason being that `:proposal` cant be blank, should I change the test?